src/test/java/seedu/address/storage/JsonAdaptedDriverTest.java

```diff
@@ -0,0 +1,65 @@
+package seedu.address.storage;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static seedu.address.storage.JsonAdaptedDriver.MISSING_FIELD_MESSAGE_FORMAT;
+import static seedu.address.testutil.Assert.assertThrows;
+import static seedu.address.testutil.TypicalDrivers.DRIVER_BENSON;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.model.person.Name;
+import seedu.address.model.person.Phone;
+import seedu.address.model.person.driver.Driver;
+import seedu.address.testutil.DriverBuilder;
+
+public class JsonAdaptedDriverTest {
+    private static final String INVALID_NAME = "R@chel";
+    private static final String INVALID_PHONE = "+651234";
```

Comment:
could we use the invalid names and invalid phone number constants that we have defined in the test utils?

Reply:
Good suggestion! The invalid names and phone number currently in `CommandTestUtil` are with prefixes, perhaps I can create constants without prefixes and rename existing ones for less confusion?